Design.md

```diff
@@ -0,0 +1,52 @@
+# Design
+ 
+## Senior UI/UX Designer
+- I have at least 5 years' experience
```

Comment:
1. Replace 5 years' experience with a range - e.g. 2-4 years' experience as per https://www.glassdoor.com/Career/senior-user-experience-designer-career_KO0,31.htm or 3-5 years' experience as per https://uxdesign.cc/the-truth-about-becoming-a-senior-ux-designer-649d23ee9b32

Reply:
Done, changed to 3-5 years' experience